cmd/grpcServer/main.go

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+
+	"github.com/sigstore/rekor-monitor/internal/monitorGRPC"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	port := "9000"
+	if val, ok := os.LookupEnv("GRPC_PORT"); ok {
+		port = val
+	}
+
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%s", port))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	grpcServer := grpc.NewServer()
+	monitorGRPC.RegisterMonitorServiceServer(grpcServer, monitorGRPC.NewServer())
```

Comment:
Will we not need access to the newly instantiated `monitorGRPC.NewServer()` at some point in the future for the argument `srv MonitorServiceServer`?

Reply:
Yes, that would be better